test/hello-world.pkr.hcl

```diff
@@ -1,8 +1,17 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+packer {
```

Comment:
Why does this template need the amazon plugin? It just uses the file builder right?

Reply:
I suggest using https://github.com/hashicorp/packer-plugin-hashicups or another similar test plugin instead of loading in the Amazon plugin, as it'll be lighter wait, I'd prefer to not end up testing cloud provider logic here, as we already test that in the plugin acceptance tests themselves